src/mixtures/mixturemodel.jl

```diff
@@ -386,7 +386,7 @@ function _mixlogpdf!(r::AbstractArray, d::AbstractMixtureModel, x)
     p = probs(d)
     @assert length(p) == K
     n = length(r)
-    Lp = Matrix{eltype(x)}(undef, n, K)
+    Lp = Matrix{eltype(p)}(undef, n, K)
     m = fill(-Inf, n)
     for i = 1:K
```

Comment:
shouldn't `@inbounds` be on the for statement?

Reply:
also `for i in eachindex(p)` is faster

Reply:
> shouldn't @inbounds be on the for statement?

Hah. I guess that's a historical typo. Nice catch.

> also for i in eachindex(p) is faster

 Got it.

Reply:
That's the same thing as long as `@inbounds` is applied to all `getindex` calls. Also, `eachindex` isn't faster for `Array`, but it's more generic.

Reply:
good catch, I always thought Base.OneTo(n) was faster